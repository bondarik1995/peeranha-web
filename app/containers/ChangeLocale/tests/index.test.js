// import React from 'react';
// import { shallow } from 'enzyme';

// import { ChangeLocale } from '../index';

describe('<ChangeLocale />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
