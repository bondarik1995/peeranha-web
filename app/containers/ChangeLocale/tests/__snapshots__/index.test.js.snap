// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChangeLocale /> should match the snapshot 1`] = `
<span
  className="dropdown"
>
  <button
    aria-expanded="true"
    aria-haspopup="true"
    className="dropdown-toggle"
    data-toggle="dropdown"
    id="dropdownMenu1"
    type="button"
  >
    <small>
      Lang
    </small>
    <span
      className="caret"
    />
  </button>
  <ul
    aria-labelledby="dropdownMenu1"
    className="dropdown-menu p-0"
  >
    <li
      className="pl-2"
      onClick={[Function]}
      role="presentation"
    >
      en
    </li>
    <li
      className="pl-2"
      onClick={[Function]}
      role="presentation"
    >
      ru
    </li>
  </ul>
</span>
`;
